sklearn/calibration.py

```diff
@@ -63,8 +63,8 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin):
 
         - None, to use the default 3-fold cross-validation,
         - integer, to specify the number of folds.
-        - An object to be used as a cross-validation generator.
-        - An iterable yielding train/test splits.
+        - :term:`CV splitter <CV splitter>`,
```

Comment:
You still shouldn't have that expression in angle brackets. It is unnecessary when the text to display is the same as the link target name